src/listeners/messageCreate.ts

```diff
@@ -38,7 +39,7 @@ const punishSpammersAndTrolls = async (message: Message): Promise<boolean> => {
     await message.delete();
     if (message.member) {
       const user = message.member.user;
-      const reason = 'Spammer/troll/got hacked';
+      const reason = 'Spammer/troll/got hacked'+'for appeal, please contact: '+APPEAL_MODERATOR_ID;
```

Comment:
make sure this looks right when you test it, there doesn't seem to be a space between hacked and for